documentation/src/main/asciidoc/userguide/chapters/batch/Batching.adoc

```diff
@@ -398,7 +402,19 @@ To use this strategy, you need to configure the following configuration property
 />
 ----
 
-Now, when running the previous test case, Hibernate generates the following SQL statements:
+NOTE: We strongly recommend the use of the first option, i.e. manually adding the temporary tables, because the inline strategy is set to be removed in a future release. Also, there is no equivalent strategy for inserts.
+
+IMPORTANT: Additionally, automatic creation of temporary tables should be deactivated. This is done by setting the +
+ +
+`hibernate.query.mutation_strategy.global_temporary.create_tables` and `hibernate.query.mutation_strategy.global_temporary.drop_tables` +
+ +
+*or* +
+ +
+`hibernate.query.mutation_strategy.persistent.create_tables` and `hibernate.query.mutation_strategy.persistent.drop_tables` +
+ +
+properties (depending on the default strategy for the dialect) to `false`
+
+With the inline strategy, when running the previous test case, Hibernate generates the following SQL statements:
```

Comment:
```suggestion
IMPORTANT: We strongly recommend the use of the first option, i.e. manually adding the temporary tables, because the inline strategy is set to be removed in a future release. Also, there is no equivalent strategy for inserts.
+
Additionally, automatic creation of temporary tables should be deactivated. This is done by setting the +
 +
`hibernate.query.mutation_strategy.global_temporary.create_tables` and `hibernate.query.mutation_strategy.global_temporary.drop_tables` +
 +
*or* +
 +
`hibernate.query.mutation_strategy.persistent.create_tables` and `hibernate.query.mutation_strategy.persistent.drop_tables` +
 +
properties (depending on the default strategy for the dialect) to `false`

With the inline strategy, when running the previous test case, Hibernate generates the following SQL statements:
```

Or you could even use the following syntax to out of doing the annoying + signs:

```
[IMPORTANT]
====
Your stuff here...

No need for + signs
====
```